front-page.php

```diff
@@ -1,9 +1,27 @@
 <?php
 get_header();
-?>
 
+// Get splash image
+$splash_images = get_post_meta($post->ID, '_igv_home_splash_images', true);
```

Comment:
I've stated before that I like this pattern better. What do you think @mikeyrayvon?